.travis.yml

```diff
@@ -22,7 +22,7 @@ install:
   - source /tmp/.nvm/nvm.sh
   - nvm install $NODE_VERSION
   - nvm use --delete-prefix $NODE_VERSION
-  - npm install -g npm@5.1.0
```

Comment:
I'm not sure these should be strictly necessary but I also think consistency in the `npm` used to install the script dependencies and the `npm` used by the remainder of the scripts to install everything else, but it also couldn't hurt.